src/test/java/core/basesyntax/model/FruitTransactionTest.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax.model;
+
+import static core.basesyntax.model.FruitTransaction.Operation.convertor;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+class FruitTransactionTest {
```

Comment:
do you really need tests for your model?

Reply:
I have tested the Operation enum, which is a nested class in FruitTransactionTest